docs/api.yml

```diff
@@ -533,3 +533,12 @@ components:
               userID:
                 type: string
                 example: "6620cf87-44ed-456e-9798-d5cbeb7fa10a"
+        images:
+          type: object
+          properties:
+            card:
+              type: string
+              example: <url link>
```

Comment:
We should only include actual examples, so these two properties should not have any examples